apps/admin/src/pages/hackathons/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+
+import { Button } from "~/components/ui/button";
+import { Sheet, SheetContent } from "~/components/ui/sheet";
+import { displayHackathons } from "./displayHackathons";
+import { AddHackathonForm } from "./hackathons";
+
+export function Hackathons() {
+  const [addHackathonFormSheetOpen, setAddHackathonFormSheetOpen] =
+    useState(false);
+
+  return (
+    <div className="px-4">
+      <div className="mx-auto max-w-5xl">
+        <div className="left-2 mb-4">
+          <Button onClick={() => setAddHackathonFormSheetOpen(true)}>
+            Add Hackathon
+          </Button>
+        </div>
+      </div>
+      {displayHackathons()}
```

Comment:
Like the prev comment about this function, please just use `HackathonsTable`.

Reply:
Ok